mut/stage.py

```diff
@@ -84,6 +85,33 @@
 T = TypeVar('T')
 
 
+class CacheControl:
+    """A manager for cache-control headers. When constructed it takes a list of stanzas,
+       each of the type Tuple[List[str], str]: this maps a set of shell-style wildcards
+       to a cache-control header. See the DEFAULT_PATTERN for an example.
+    """
+    # Based on https://www.gatsbyjs.com/docs/caching/
+    DEFAULT_PATTERN = '''[
+        [["*.html", "*/page-data/*.json", "*/sw.js"],
```

Comment:
We need to apply this rule to JSON files in various directories under `page-data`; should the rule be `*/page-data/**/*.json`? ([source](https://www.gatsbyjs.com/docs/caching/#page-data))

Reply:
Nope -- these are matched as strings, not filenames, as per the Python `fnmatch` docs, so `/` is not special. I can clarify that.

Reply:
Fascinating! Thanks 😄